docs/reference/commandline/dockerd.md

```diff
@@ -780,6 +785,113 @@ conditions the user can pass any size less then the backing fs size.
 $ sudo dockerd -s overlay2 --storage-opt overlay2.size=1G
 ```
 
+
+#### Windowsfilter options
+
+##### `size`
+
+Specifies the size to use when creating the sandbox which is used for containers.
+Defaults to 20G.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt size=40G
+```
+
+#### LCOW (Linux Containers on Windows) options
+
+##### `lcow.globalmode`
```

Comment:
These are all the supported daemon options for startup. For run, with the exception of globalmode, they would all be supported.  



> Are there use cases of having all these configurable?

Absolutely. A user may want to run different containers with different kernel revisions as one such example.